### DISCUSSION TITLE
Non-grid thumbnail display

### DISCUSSION THREAD

**Discussion by fcimeson** [photoprism/photoprism#2255] [Ideas]
Offer an option for thumbnail previews other than a uniform gird. This type of vi.ew is provided by many other services such as google photos.

![non-grid](https://user-images.githubusercontent.com/3228114/163564305-05a6fe64-3665-49ad-aebf-61353be0bc5a.png)

**Comment by graciousgrey** [comment]
We already have an open issue for a view like this: https://github.com/photoprism/photoprism/issues/85 :)

**Reply by fcimeson** [reply]
Thanks!
